tests/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ set(tests
   test_srcfd
   test_stream_timeout
   test_xpub_manual
+  test_xpub_topic
```

Comment:
please also add this to `Makefile.am`

Reply:
I hope the changes to Makefile.am are OK. I'm not familiar with automake...